### ISSUE TITLE
For the ishlist 

### ISSUE THREAD

**Issue by tomk3003** [takezoe/gitbucket-ci-plugin#62]

**Comment by takezoe** [comment]
Do you mean the the build script configured by GitBucket admin is used in all repositopries?

**Comment by tomk3003** [comment]
Yes, then the script could read a config file from the repository and the admin  can define what can be run. This would be more secure than letting everyone run their own script.
